og.module

```diff
@@ -1816,12 +1816,16 @@ function og_membership_access($op, $entity, $account = NULL, $entity_type = 'og_
  *   The entity object, or the entity ID.
  *
  * @return
- *   TRUE or FALSE if the entity is a group.
+ *   TRUE if the entity exists and is a group, FALSE otherwise.
  */
 function og_is_group($entity_type, $entity) {
   if (is_numeric($entity)) {
     $entity = entity_load_single($entity_type, $entity);
   }
+  if (!$entity) {
```

Comment:
that is "baby sitting" I prefer not add. The API function is expecting to get an object or an entity ID. If you didn't pass it, I prefer it to throw you an exception.